docs/content/storage-drivers/s3.md

```diff
@@ -32,6 +33,9 @@ Amazon S3 or S3 compatible services for object storage.
 | `usedualstack` | no | Use AWS dual-stack API endpoints. |
 | `accelerate` | no | Enable S3 Transfer Acceleration. |
 | `objectacl`  | no | The S3 Canned ACL for objects. The default value is "private". |
+| `usedualstack` | no | Use AWS S3 dual-stack endpoints, which support both IPv6 and IPv4, when set to `true`. The default is `false`. |
+| `multipartcombinesmallpart` | no | Combines small pending uploads with the ready part when `true`. The default is `true`. |
```

Comment:
Where is this option implemented? I don't see it anywhere else in the PR nor could I find any evidence in the code base that this was already implemented.

Reply:
The option was removed in https://github.com/distribution/distribution/pull/3556